packages/dredd/test/unit/sortTransactions-test.ts

```diff
@@ -0,0 +1,104 @@
+import R from 'ramda';
+import { expect } from 'chai';
+import sortTransactions from '../../lib/sortTransactions';
+import { Transaction, RESTMethod } from '../../lib/general';
+
```

Comment:
I don't see a similar test written in JS being removed. Does it mean you want to keep it to be sure nothing got broken, or that it doesn't exist and this part isn't tested?

Reply:
I couldn't find any existing test that would cover `sortTransactions`. To my knowledge this adds the missing unit test, so there is no JavaScript predecessor to remove. Please, point me if you know where this behavior may be tested already. Thanks.